client/rest-high-level/src/test/java/org/elasticsearch/client/RestHighLevelClientTests.java

```diff
@@ -199,29 +199,32 @@ public ActionRequestValidationException validate() {
 
         {
             ActionRequestValidationException actualException = expectThrows(ActionRequestValidationException.class,
-                    () -> restHighLevelClient.performRequest(request, null, RequestOptions.DEFAULT, null, null));
+                    () -> restHighLevelClient.getRequestActions().performRequest(request, null, RequestOptions.DEFAULT, null, null));
```

Comment:
Should these be in a `RestRequestActionsTests`?

Reply:
good catch on splitting this up. incoming shortly. this also gets rid of a lot of uses of the `getRequestActions`, which leads me to wonder if we should have that method at all :)